From: "Barak A. Pearlmutter" <barak+git@pearlmutter.net>
Date: Sun, 30 Nov 2025 19:40:48 +0000
Subject: remove readme.txt from installation list

---
 doc/Makefile.am | 2 +-
 1 file changed, 1 insertion(+), 1 deletion(-)

diff --git a/doc/Makefile.am b/doc/Makefile.am
index 3e348b2..ccd779f 100644
--- a/doc/Makefile.am
+++ b/doc/Makefile.am
@@ -8,7 +8,7 @@ EXTRA_DIST += pstoedit.pdf pstoedit.htm pstoedit.1 latex2man.local latex2man.css
 #
 man_MANS = pstoedit.1
 
-doc_DATA = pstoedit.pdf pstoedit.htm readme.txt changelog.htm 
+doc_DATA = pstoedit.pdf pstoedit.htm changelog.htm 
 ## Unfortunately this is broken:
 #info_TEXINFOS = pstoedit.texi
 
