From: "Barak A. Pearlmutter" <barak+git@pearlmutter.net>
Date: Thu, 2 Sep 2021 11:53:42 +0100
Subject: spelling

---
 src/drvbase.cpp | 2 +-
 1 file changed, 1 insertion(+), 1 deletion(-)

diff --git a/src/drvbase.cpp b/src/drvbase.cpp
index 4e08978..ef12978 100644
--- a/src/drvbase.cpp
+++ b/src/drvbase.cpp
@@ -937,7 +937,7 @@ void drvbase::dumpRearrangedPaths()
 	// Rearrange the path if necessary
 	if ((numpaths > 1) && (currentLineWidth() == 0.0) && (currentShowType() != drvbase::stroke)) {
 		if (verbose)
-			errf << "Starting rearrangment of subpaths" << endl;
+			errf << "Starting rearrangement of subpaths" << endl;
 		outputPath->rearrange();
 		numpaths = nrOfSubpaths();
 	}
